### PR TITLE
github actions: add clean up to master too

### PR DESCRIPTION
**Pull request by medyagh**

### CODE REVIEW

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -176,6 +176,17 @@ jobs:
           echo $env:computerName
           ls 
           $ErrorActionPreference = "SilentlyContinue"
+          cd minikube_binaries
+          ls
+          $env:KUBECONFIG="${pwd}\testhome\kubeconfig"
+          $env:MINIKUBE_HOME="${pwd}\testhome"
+          .\minikube-windows-amd64.exe delete --all --purge
+          Get-VM | Where-Object {$_.Name -ne "DockerDesktopVM"} | Foreach {
```
Comment (tstromberg) [inline .github/workflows/master.yml:184]: This is very strange and needs a comment.

File: .github/workflows/master.yml [hunk 2]
```diff
@@ -176,6 +176,17 @@ jobs:
...
+            minikube-windows-amd64.exe delete -p $_.Name
```
Comment (tstromberg) [inline .github/workflows/master.yml:185]: I think this line can be removed.

File: .github/workflows/master.yml [hunk 3]
```diff
@@ -176,6 +176,17 @@ jobs:
...
+            Suspend-VM $_.Name
```
Comment (tstromberg) [inline .github/workflows/master.yml:186]: Suspend-VM is meaningless if Stop-VM is called subsequently.

Comment (medyagh) [reply .github/workflows/master.yml:186]: I copied from our jenkoins one

File: .github/workflows/master.yml [hunk 4]
```diff
@@ -176,6 +176,17 @@ jobs:
...
+            Stop-VM $_.Name -Force
```
Comment (tstromberg) [inline .github/workflows/master.yml:187]: For clarity, I think this should specify `-Name`, as per https://docs.microsoft.com/en-us/powershell/module/hyper-v/stop-vm?view=win10-ps

I think `-TurnOff` may be a valuable flag as well.

File: .github/workflows/master.yml [hunk 5]
```diff
@@ -176,6 +176,17 @@ jobs:
...
+            Remove-VM $_.Name -Force
+          }
           Remove-Item minikube_binaries -Force -Recurse
```
Comment (tstromberg) [inline .github/workflows/master.yml:190]: A previous step ran `cd minikube_binaries` -- does that mean there is a minikube_binaries within it, or do we need to remove ourselves from that directory first?

File: .github/workflows/master.yml (hunk 2 above)
Comment (medyagh) [reply .github/workflows/master.yml:185]: sometimes it is better to do that if delete --all is not doing the right thing

**Pull request merged**